common/src/main/java/com/wynntils/features/user/overlays/ShamanTotemTrackingFeature.java

```diff
@@ -0,0 +1,202 @@
+import com.wynntils.core.features.overlays.sizes.GuiScaledOverlaySize;
+import com.wynntils.gui.render.FontRenderer;
+import com.wynntils.gui.render.HorizontalAlignment;
+import com.wynntils.gui.render.TextRenderSetting;
+import com.wynntils.gui.render.TextRenderTask;
+import com.wynntils.gui.render.VerticalAlignment;
+import com.wynntils.mc.event.RenderEvent;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.utils.StringUtils;
+import com.wynntils.wynn.event.TotemRemovedEvent;
+import com.wynntils.wynn.event.TotemSummonedEvent;
+import com.wynntils.wynn.objects.ShamanTotem;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.world.entity.decoration.ArmorStand;
+import net.minecraft.world.scores.PlayerTeam;
+import net.minecraft.world.scores.Scoreboard;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ShamanTotemTrackingFeature extends UserFeature {
+    @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
+    private final ShamanTotemTimerOverlay shamanTotemTimerOverlay = new ShamanTotemTimerOverlay();
+
+    @Config
+    public boolean highlightShamanTotems = true;
+
+    @Config
+    public static ChatFormatting totem1Color = ChatFormatting.WHITE;
+
+    @Config
+    public static ChatFormatting totem2Color = ChatFormatting.AQUA;
+
+    @Config
+    public static ChatFormatting totem3Color = ChatFormatting.RED;
+
+    private static final String TOTEM_HIGHLIGHT_TEAM_BASE = "wynntilsTH";
+
+    @SubscribeEvent
+    public void onTotemSummoned(TotemSummonedEvent e) {
+        if (!highlightShamanTotems) return;
+
+        int totemNumber = e.getTotemNumber();
+        ArmorStand totemAS = e.getTotemEntity();
+
+        ChatFormatting color =
+                switch (totemNumber) {
+                    case 1 -> totem1Color;
+                    case 2 -> totem2Color;
+                    case 3 -> totem3Color;
+                    default -> throw new IllegalArgumentException(
+                            "totemNumber should be 1, 2, or 3! (color switch in #onTotemSummoned in ShamanTotemTrackingFeature");
+                };
+
+        // Make or get scoreboard to set highlight colors
+        Scoreboard scoreboard = McUtils.mc().level.getScoreboard();
+        if (!scoreboard.getTeamNames().contains(TOTEM_HIGHLIGHT_TEAM_BASE + totemNumber)) {
+            scoreboard.addPlayerTeam(TOTEM_HIGHLIGHT_TEAM_BASE + totemNumber);
+        }
+        PlayerTeam team = scoreboard.getPlayerTeam(TOTEM_HIGHLIGHT_TEAM_BASE + totemNumber);
+        team.setColor(color);
+
+        scoreboard.addPlayerToTeam(totemAS.getStringUUID(), team);
+        totemAS.setSharedFlag(6, true); // Makes the totem glow
+    }
+
+    @SubscribeEvent
+    public void onTotemDestroy(TotemRemovedEvent e) {
+        if (!highlightShamanTotems) return;
+
+        // Teams should be destroyed and recreated every cast to allow the user to change totem highlight colors without
+        // having to reload the feature
```

Comment:
See my comments above. Even if we keep the scoreboard magic, I suggest you delete teams in `onConfigChange` (`Feature` override).